### ISSUE TITLE
ESS failed to run: ess-s-define-runners is void

### ISSUE THREAD

**Issue by thjwong** [vigou3/emacs-modified-windows#9]
I run into the problem 
> ess-s-define-runners is void

when trying to start R with M-x R

This thread mentioned it as a recent known bug and updating to latest ESS version should solve the problem.
[https://emacs.stackexchange.com/questions/42339/ess-failed-to-run-ess-s-define-runners-is-void](https://emacs.stackexchange.com/questions/42339/ess-failed-to-run-ess-s-define-runners-is-void)

The related bug 596 in ESS:
[https://github.com/emacs-ess/ESS/issues/596](https://github.com/emacs-ess/ESS/issues/596)

**Comment by vigou3** [comment]
I cannot reproduce the problem with 26.1-modified-2 (which includes the latest stable release of ESS) and R 3.5.1 on Windows 7.

The bug you are referring to is triggered by installing a version of ESS from MELPA on top of a "system" install. If it's what you are doing then you're on your own since my distribution only ships with official releases of packages.

**Comment by thjwong** [comment]
Thanks for the prompt reply.
I removed the ESS (ess-20180723.1428) from the MELPA folder and R (3.5.1) now starts without any problem. Before doing that I had to replace all the lisp files in the C:\Program Files\GNU Emacs 26.1\share\emacs\site-lisp\ess folder and can only use R 3.4.4 because of an issue of the stringi package in R 3.5.1

I haven't replaced the original versions of lisp back to the site-lisp\ess folder yet, but I believe that it should work.
